src/landing/LoginForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import {
+  Button, Checkbox,
+  FormControl, FormControlLabel, FormHelperText, Input, InputLabel, Typography,
+} from '@material-ui/core';
+import './styles.css';
+
+const LoginForm = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const [passwordFieldType, setPasswordFieldType] = useState('password');
+
+  const changePasswordFieldType = () => {
+    if (passwordFieldType === 'password') setPasswordFieldType('text');
+    else setPasswordFieldType('password');
```

Comment:
While this is correct, it is a good practice to always add the { } for the if-else statements, even if we have 
only one instruction.